### PR TITLE
feat: bring `prebundleSvelteLibraries` out of experimental

### PR DESCRIPTION
**Pull request by benmccann**
Hurray!

We can address https://github.com/sveltejs/vite-plugin-svelte/issues/421 in a follow-up since it only makes prebundling less effective, but doesn't break any projects

### CODE REVIEW

File: packages/vite-plugin-svelte/src/index.ts
Comment (bluwy) [inline packages/vite-plugin-svelte/src/index.ts:88]: When resolving the v-p-s options, should we auto-set `options.prebundleSvelteLibraries` if `options.experimental.prebundleSvelteLibraries` is set, and a warning? Not necessarily needed since it was an experimental option, but would be a nice-to-have for existing users who turn this on.

Comment (benmccann) [reply packages/vite-plugin-svelte/src/index.ts:95]: yes, good idea. I've added that to the `config` hook

**Review by bluwy — APPROVED** [review]
Added a commit to refactor a bit of stuff 😅 but this looks good! Waiting for dominik's approval too just in case.

File: packages/vite-plugin-svelte/src/utils/options.ts
Comment (dominikg) [inline packages/vite-plugin-svelte/src/utils/options.ts:296]: This should be thrown as an error. Otherwise people update v-p-s and suddenly svelte dependencies are no longer prebundled and if they don't check warnings that might have scrolled by quickly, they'll miss it.

**Comment by dominikg** [comment]
If you feel comfortable promoting it out of experimental i'm ok with it too. One thing that can be a bit of a pain still is that if it discovers new svelte components while navigating an app during dev, prebundling runs again to include them unless you add all of them to optimizeDeps.include (which has to be done explicitly because there is no glob support in that (why not?)

**Comment by bluwy** [comment]
I think it would only re-prebundle when it discovers a new deep import, but this PR/feature should bring it closer to a Vite issue so maybe we can get glob support for Vite 4 too 🤔 Ref: https://github.com/vitejs/vite/issues/5419

**Pull request merged**
